tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -48,55 +42,10 @@ def rows_to_csv(rows):
     return output.getvalue()
 
 
-@pytest.mark.parametrize("reference", [None, "reference_from_client"])
-def test_post_sms_notification_returns_201(notify_api, client, sample_template_with_placeholders, mocker, reference):
```

Comment:
We might want to change this test instead of axing it, by mocking the celery database-task and checking what it receives maybe?